app/utils/common.js

```diff
@@ -231,6 +233,12 @@ export const fetchParamsFromURL = (location, paramName, names) => {
   );
 };
 
+export const removePageProgressFromStorage = (key) => {
```

Comment:
Are we using localStorage for storing form progress?  I think that is commented out.  If so you can comment out the call to this method